### ISSUE TITLE
request without body are automatically invalidated with 400

### ISSUE THREAD

**Issue by vejmoj1** [Ge0rg3/flask-parameter-validation#7]
Seems that requests without body are automatically resulted as bad requests but according to REST standard body is optional part. Even GET requests without body are failing and this is quite common case.

Seems that the following update in `parameter_validation.py` is fixing this issue.  Method `nested_func` star on line 16 currently.
```python
        def nested_func(**kwargs):
            # Step 1 - Combine all flask input types to one dict
            request_inputs = {
                Route: kwargs.copy(),
                Json: request.json if request.is_json else {},
                Query: request.args.to_dict(),
                Form: request.form.to_dict(),
                File: request.files.to_dict()
            }
```

**Comment by Ge0rg3** [comment]
Oh yikes that's definitely not supposed to happen.. Thanks for letting me know! Will fix during Hacktoberfest 👌 @vejmoj1

**Comment by Ge0rg3** [comment]
Thank you @vejmoj1, can't believe this issue was there for so long! Just fixed in #8 😄

**Comment by vejmoj1** [comment]
Thank you @Ge0rg3 for fixing the issue. It was pretty surprising when I have wondered what went wrong 😅
